Neuro/Tasks/TasksHandler.cs

```diff
@@ -73,6 +73,64 @@ public void UpdatePathToTask(PlayerTask task = null)
         }
     }
 
+    [HideFromIl2Cpp]
+    public PlayerTask GetClosestTask()
+    {
+        PlayerTask closestTask = null;
+        float closestDistance = Mathf.Infinity;
+
+        foreach (PlayerTask t in PlayerControl.LocalPlayer.myTasks)
+        {
+            if (!t.IsComplete && t.HasLocation)
+            {
+                Vector2[] path = NeuroPlugin.Instance.Pathfinding.FindPath(PlayerControl.LocalPlayer.transform.position, t.Locations[0]);
```

Comment:
Use `.At()` extension method
```suggestion
                Vector2[] path = NeuroPlugin.Instance.Pathfinding.FindPath(PlayerControl.LocalPlayer.transform.position, t.Locations.At(0));
```